forc-test/src/lib.rs

```diff
@@ -683,11 +683,11 @@ pub fn decode_log_data(
         .logged_types
         .iter()
         .flatten()
-        .map(|logged_type| (logged_type.log_id.as_str(), logged_type.application.clone()))
+        .map(|logged_type| (logged_type.log_id.clone(), logged_type.application.clone()))
```

Comment:
while rebasing accepted @esdrubal's changes blindly. I am reverting this one to mine, as it does not clone